### ISSUE TITLE
about base_lr

### ISSUE THREAD

**Issue by tiger1933** [wangbm/MTCNN-Tensorflow#10]
When I begin to train net not fine-tuning. How is the learning rate set?Can anyone help me! thanks
